integrations/access/opsgenie/client.go

```diff
@@ -128,21 +129,28 @@ func NewClient(conf ClientConfig) (*Client, error) {
 		}}).
 		SetHeader("Authorization", "GenieKey "+conf.APIKey).
 		SetBaseURL(conf.APIEndpoint)
+
+	client.OnAfterResponse(common.OnAfterResponse("opsgenie", errWrapper, conf.StatusSink))
```

Comment:
Nit: Prefer to use existing constants:
```suggestion
	client.OnAfterResponse(common.OnAfterResponse(types.PluginTypeOpsgenie, errWrapper, conf.StatusSink))
```